pkg/Tizen.NET.API10/build/Tizen.NET.API10.targets

```diff
@@ -9,12 +9,15 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Reference Include="$(MSBuildThisFileDirectory)\ref\*.dll">
+    <Reference Include="$(MSBuildThisFileDirectory)tizen10.0\ref\*.dll">
       <Private>false</Private>
       <Visible>false</Visible>
       <NuGetPackageId>Microsoft.NETCore.App.Ref</NuGetPackageId>
       <NuGetPackageVersion>$(_TizenNETCoreAppPackageVersion)</NuGetPackageVersion>
     </Reference>
   </ItemGroup>
 
+  <ImportGroup Condition=" '$(ExcludeRestorePackageImports)' != 'true' ">
+    <Import Project="$(MSBuildThisFileDirectory)..\buildTransitive\Tizen.NUI.XamlBuild.targets" Condition="Exists('$(MSBuildThisFileDirectory)..\buildTransitive\Tizen.NUI.XamlBuild.targets')" />
+  </ImportGroup>
 </Project>
```

Comment:
this is trivial thing but could you add new line at the end of file?
- reference: https://stackoverflow.com/questions/729692/why-should-text-files-end-with-a-newline